ops/ccs-ops-misc/load_test/v1/eob_test_id_lastUpdated.py

```diff
@@ -0,0 +1,45 @@
+import urllib3
+import common.config as config
+import common.data as data
+import common.errors as errors
+import common.test_setup as setup
+import common.validation as validation
+from locust import HttpUser, task, events
+
+server_public_key = setup.loadServerPublicKey()
+setup.disable_no_cert_warnings(server_public_key, urllib3)
+
+eob_ids = data.load_bene_ids()
+client_cert = setup.getClientCert()
+setup.set_locust_env(config.load())
+last_updated = data.get_last_updated()
+
+class BFDUser(HttpUser):
+    @task
+    def explanation_of_benefit(self):
+        if len(eob_ids) == 0:
+            errors.no_data_stop_test(self)
+
+        id = eob_ids.pop()
+        response = self.client.get(f'/v1/fhir/ExplanationOfBenefit?_lastUpdated=gt{last_updated}&patient={id}&_format=json',
+                cert=client_cert,
+                verify=server_public_key,
+                name='/v1/fhir/ExplanationOfBenefit search by id / lastUpdated')
+
+'''
+Adds a global failsafe check to ensure that if this test overwhelms the
+database, we bail out and stop hitting the server.
+'''
+@events.init.add_listener
+def on_locust_init(environment, **_kwargs):
+    validation.setup_failsafe_event(environment, validation.SLA_EOB_WITHOUT_SINCE)
+
+'''
+Adds a listener that will run when the test ends which checks the various
+response time percentiles against the SLA for this endpoint.
+'''
+@events.test_stop.add_listener
+def on_locust_quit(environment, **_kwargs):
+    validation.check_sla_validation(environment, validation.SLA_EOB_WITHOUT_SINCE)
```

Comment:
should these be using SLA_EOB_WITH_SINCE?

Reply:
as far as I can tell, using lastUpdated is technically different from using since in fhir terms? Unless that is our implementation of since? I added the SLAs for WITH_SINCE, but i didnt see anyone actually use _since explicitly in the splunk logs.

Reply:
SLAs are from this confluence page that someone pointed me at when we discussed it in a meeting: https://confluence.cms.gov/pages/viewpage.action?pageId=292192505